### ISSUE TITLE
Should we update the Wildfly 10 quickstart to latest WF (match overlay distro)? Discuss!

### ISSUE THREAD

**Issue by msavy** [apiman/apiman#983]
Currently https://github.com/apiman/apiman-community-quickstarts has a WF 10 ES overlay (which I'm currently updating to the latest versions of Apiman and to allow newer versions of ES to start, etc). 

I reckon we should align that version with the overlay in the main repo. May as well keep them in lockstep. Given that WF doesn't really do patch releases, it don't think we need to maintain support for really old versions.

I suspect the 10 part is probably a bit of a misnomer by now anyway.

Anyone object?

**Comment by msavy** [comment]
BTW, I know the Tomcat distro is popular with some (e.g. @outofcoffee  and @volkflo use it, I believe?). 

I don't know whether it's worth bringing that back into the main distro, or just to continue releasing it from the quickstarts?

Certainly, we are prepared to maintain that and ensure it all works nicely.

I'm currently working on ensuring everything is fine for newer Tomcat 8.x versions (which all ties in the the work in the first comment, of course).

**Comment by volkflo** [comment]
Yeah, we discussed this a while ago (https://github.com/apiman/apiman/pull/892). 
I would be happy to bring tomcat back to life (maybe also get rid of the version number behind the module name like we did it for wildfly).
But in general, I think the quickstarts/distros should be more production-ready.
Otherwise, IMO, it would be enough to just provide one simple docker image to test and try out something with apiman
